tests/tck/features/optimizer/PushFilterDownGetNbrsRule.feature

```diff
@@ -106,10 +106,10 @@ Feature: Push Filter down GetNeighbors rule
       | 2008       |
       | 2012       |
     And the execution plan should be:
-      | id | name         | dependencies | operator info                         |
-      | 0  | Project      | 1            |                                       |
-      | 1  | GetNeighbors | 2            | {"filter": "(serve.start_year<2017)"} |
-      | 2  | Start        |              |                                       |
+      | id | name         | dependencies | operator info                                                       |
+      | 0  | Project      | 1            |                                                                     |
+      | 1  | GetNeighbors | 2            | {"filter": "((serve.start_year<2017) AND (serve.start_year<2017))"} |
```

Comment:
Why need to filter same expression twice?

Reply:
Since the optimizer has not been modified, the build execution plan and the push down of the optimizer will repeat.

Reply:
better to solve the expression issue firstly. 

By the way, I'm refactoring the join related execution plan in go sentence recently, there'll be more better solution for the problem after that.

Reply:
Then I don't think the PR needs to be merged. The final solution is to push down the optimizer solution. I look forward to your early release

Reply:
> Then I don't think the PR needs to be merged. The final solution is to push down the optimizer solution. I look forward to your early release

If you think this pr doesn't need to be merged, would you like to close the pr? Thanks a lot for your contribution.

Thank you again for your work ❤️